src/client/index.ts

```diff
@@ -112,6 +113,13 @@ export interface NotificationOptions {
      */
     date?: Date;
 
+    /**
+     * The expiry date and time of the notification. If specified, the notification will be removed at this time, or as
+     * soon as possible after. If no expiry is specified, the notification will never expire, and will persist until it
+     * is closed.
+     */
+    expiry?: Date|null;
```

Comment:
I prefer `expires` as it also matches the HTTP header https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expires

Reply:
Confirmed with Phil - will go with `expires` and `onExpire`

Reply:
Terminology changed in next push

Reply:
✅